core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -870,13 +870,22 @@ private SqlLibraryOperators() {
    * INTERVAL int64 date_part)" but in Calcite the second argument can be any
    * interval expression, not just an interval literal. */
   @LibraryOperator(libraries = {BIG_QUERY})
-  public static final SqlFunction TIMESTAMP_SUB =
+  public static final SqlBasicFunction TIMESTAMP_SUB =
       SqlBasicFunction.create(SqlKind.TIMESTAMP_SUB, ReturnTypes.ARG0_NULLABLE,
           OperandTypes.TIMESTAMP_INTERVAL)
           .withFunctionType(SqlFunctionCategory.TIMEDATE);
 
-  /** The "TIMESTAMP_TRUNC(timestamp, timeUnit[, timeZone])" function (BigQuery);
-   * truncates a TIMESTAMP value to the beginning of a timeUnit. */
+  /** BigQuery's {@code DATETIME_SUB(timestamp, interval)} function
+   * is a synonym for TIMESTAMP_SUB because in Calcite, DATETIME
+   * is an alias for TIMESTAMP. */
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction DATETIME_SUB =
+      TIMESTAMP_SUB.withName("DATETIME_SUB");
+
+  /** The "TIMESTAMP_TRUNC(timestamp_expression, date_time_part[, time_zone])"
```

Comment:
Can you restore my original edits, so that the signature reads `TIMESTAMP_TRUNC(timestamp, timeUnit[, timeZone])`. `timestamp_expression` is copy-pasted from the bigQuery doc but is inconsistent with how other functions are documented in this file.